src/einsteinpy/tests/test_plotting/test_staticgeodesicplotter.py

```diff
@@ -54,3 +55,32 @@ def test_plot_save_saves_plot(mock_save):
     name = "test_plot.png"
     cl.save(name)
     mock_save.assert_called_with(name)
+
+
+@mock.patch("einsteinpy.plotting.geodesics_static.StaticGeodesicPlotter.plot_attractor")
```

Comment:
If you just have to check that every thing works fine. Why are you mocking plot attractor?

Reply:
I am mocking plot attractor two times, the reasons are mentioned below:

1)Auto scaling
Here the _draw_attractor will be called and the "if" block will be executed and not the else block.

Now since the else block is not executed till now (because attractor_radius_scale == -1.0) I will have to call it again.

2)Manual scaling
Here the _draw_attractor will be called and the "else" block will be executed .

I don't understand why I do not have to mock plot attractor. If I don't then the lines won't be executed otherwise and the coverage will go down.

Actually I need the _draw_attractor only so I have change plot_attractor to _draw_attractor.

Can you please explain how the else block will be executed if I do not pass manual scale parameter? I don't understand how I can check that everything works fine without it.

Reply:
Do you want to assert if `_draw_attractor` is being called or not?

Reply:
Or you just want the _draw_attractor to be called?

Reply:
I just want the _draw_attractor to be called

Reply:
and then I am checking if it is called or not

Reply:
Well that's what I am emphasising. 

> I am checking if it is called or not

Why do you need to check that?

Reply:
I see, so I don't really have to check that since it is being checked by other functions.
Will remove the assert line

Reply:
Done, is it fine now?

Reply:
What did you change? xD